python/lsst/meas/algorithms/computeExPsf.py

```diff
@@ -0,0 +1,120 @@
+# This file is part of meas_algorithms.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+from lsst.meas.algorithms.treecorrUtils import TreecorrConfig
+from lsst.pex.config import Config, ConfigField
+from lsst.pipe.base import Task
+import lsst.pipe.base as pipeBase
+import treecorr
+import copy
+import numpy.typing as npt
+
+
+__all__ = ("ComputeExPsfTask", "ComputeExPsfConfig")
+
+
+class ComputeExPsfConfig(Config):
+    """Config class of ComputeExPsfTask."""
+
+    treecorr = ConfigField(
+        dtype=TreecorrConfig,
+        doc="treecorr config.",
+    )
+
```

Comment:
I suspect this needs an explicit `validate` method that calls `self.treecorr.validate`. Because `TreecorrConfig` is not a `ConfigClass` of a sub task, I am fairly certain that it will never be validated.

Reply:
There also needs to be a `setDefaults` method here that sets the `min_sep`, `max_sep` and either `nbins` or `bin_size` as appropriate here corresponding to the requirements. @jmeyers314 - what was the binning we settled on finally for TEx computations?

Reply:
For the `validate`, I tried in the unit test and putting wrong value in `ComputeExPsfConfig` (`min_sep>max_sep`) for example and it triggered the `validate` from `TreecorrConfig`. Do I need to do more than that?

Reply:
I think it'd be best to put it here, given that i)  we might want to run this dynamically outside of `computeSummaryStats` and ii) it makes sense for this class to have its own defaults for `treecorr` rather than `computeSummaryStats` setting them.